src/storage/builder.rs

```diff
@@ -32,7 +32,7 @@ use super::prelude::*;
 #[derive(Default, Debug)]
 pub struct Builder {
     config: Config,
-    ioring: Option<Rio>,
+    iodriver: IODriver,
```

Comment:
`IODriver::default()` creates `Rio` inside. This is bad, because `Rio` construction is not cheep. It is better to declare `iodriver` as `Option<IODriver>` and use `unwrap_or(|| IODriver::default())` when creating the Storage

Reply:
Actually, using `default` is bad for `IoDriver`. It is better to fallback to `IoDriver::new_sync()` if driver was not specified explicitly

Reply:
So all iodrivers should have `new_sync`?